hoot-core/src/main/cpp/hoot/core/conflate/poi-polygon/PoiPolygonReviewReducer.cpp

```diff
@@ -112,13 +128,23 @@ bool PoiPolygonReviewReducer::triggersRule(ConstElementPtr poi, ConstElementPtr
   const bool polyHasType = PoiPolygonTypeScoreExtractor::hasType(poly);
   LOG_VART(polyHasType);
 
+  const int numPolyAddresses = AddressParser::hasAddress(poly, *_map);
+  const bool polyHasAddress = numPolyAddresses > 0;
+
   //if both have addresses and they explicitly contradict each other, throw out the review; don't
-  //do it if the poly has more than one address, like in many multi-use buildings
-  if (!_addressMatch && PoiPolygonAddressScoreExtractor::elementHasAddress(poi, *_map) &&
-      PoiPolygonAddressScoreExtractor::elementHasAddress(poly, *_map))
+  //do it if the poly has more than one address, like in many multi-use buildings.
+  if (!_addressMatch && AddressParser::hasAddress(poi, *_map) && polyHasAddress)
   {
-    LOG_TRACE("Returning miss per review reduction rule #1...");
-    return true;
+    //check to make sure the only address the poly has isn't the poi itself as a way node /
+    //relation member
+    if (_polyContainsPoiAsMember(poly, poi) && numPolyAddresses < 2)
+    {
+    }
+    else
+    {
+      LOG_TRACE("Returning miss per review reduction rule #1...");
+      return true;
+    }
   }
```

Comment:
This fixed a bug where poi addresses were incorrectly being counted twice...once by themselves and once as a way node that was part of the polygon.